share/availability/window.go

```diff
@@ -0,0 +1,24 @@
+package availability
+
+import "time"
+
+const (
+	RequestWindow = 30 * 24 * time.Hour
+	StorageWindow = RequestWindow + time.Hour
+)
+
+type Window time.Duration
+
+func (w Window) Duration() time.Duration {
+	return time.Duration(w)
+}
```

Comment:
It feels like the only reason for availability.Window to be typed is nodebuilder. Perhaps it can be moved from `share` pkg to `nodebuilder/share` pkg. 

It also seems the type is used to pass not any availability.Window, but just StorageWindow, so if moved, it can be renamed to be share.StorageWindow to be more specific.

Reply:
this would cause an import cycle issue inside of nodebuilder, so would prefer not to spend time on this. Eventually, pruner will be moved into share pkg which will resolve the dependency issue.

Reply:
It is used inside nodebuilder, so it should not be a problem to move it there without cycle dependancy. Perhaps better package for it might be nodebuilder/pruner